NOTICE.txt

```diff
@@ -0,0 +1,9 @@
+Copyright © 2002-2022 Neo4j, Inc. (referred to
```

Comment:
Will this not need to go in the `graphql-ui` directory (perhaps in addition to at the root)? Will it not need to be in our packages?

Reply:
Yes that's a good question. I was unsure where to put it. But It seems a better fit in the `graphql-ui` directory. Will move it.